user/build-matrix.md

```diff
@@ -216,6 +216,43 @@ script: env $EXTRA_TESTS ./test.py $TEST_SUITE
 ```
 {: data-file=".travis.yml"}
 
+### Explicitly included jobs with only one element in the build matrix
+
+As a special case, if your build matrix has only one element _and_ you specified
+explicitly included jobs, matrix expansion is not done and the explicit jobs
+_completely_ define your build. For example:
+
+```yaml
+language: python
+python:
+  - '3.5'
+matrix:
+  include:
+    - env: EXTRA_TESTS=true
+# only defines one job with `python: 3.5` and `env: EXTRA_TESTS=true`
```

Comment:
```suggestion
# defines one job with `python: 3.5` and `env: EXTRA_TESTS=true`
```

Reply:
"Only" is here for a reason. This example is a "typical mistake" example -- its intention, implied from identifiers, is contrary to the result. And the next example is the fix for it.
So the phrasing emphasises that this is a counterintuitive result, drawing the reader's attention to this fact.

I'm not feeling too strongly about this though so I'm okay if you feel there's no need going the extra mile helping the reader with narrative devices.